docs/extras/project-ideas/index.md

```diff
@@ -0,0 +1,176 @@
+
+Probabilistic matching will allow us to build up a knowledge base of likely matches across trials based on similarities on key data fields.
+
+#### How
+
+Write one or many [processors](https://github.com/opentrials/processors) that do matching across all trial records using probabilistic techniques.
+
+#### Skills
+
+- Advanced data science/wrangling
+- Intermediate Python
+
+### Basic outlier detection
+
+#### What
+
+There is much that can potentially be learned just by looking for outliers in a large dataset. Even without specific domain knowledge, building a set of queries to find outliers can be a first step in asking questions about the data. As a starting point, consider such possibilities around participation, location, age range or interventions for a particular condition.
+
+#### How
+
+Using the [OpenTrials query interface](https://query.opentrials.net/), build a set of queries designed for outlier detection across key data fields.
+
+- [A simple introduction to outlier detection with SQL](https://www.periscopedata.com/blog/outlier-detection-in-sql.html)
+
+#### Skills
+
+- Intermediate SQL
+
+### Visualisation of discrepancies
+
+#### What
+
+By collecting a significant amount of publicly available data on clinical trials, we've found that different records for the same trial often have different information. Such differences can be, for example, in the supposed number of participants who took part in a trial, whether the trial is still ongoing, or if the trial's results have been published.
+
+These discrepancies are important as all these public sources of trial data can be taken as "official", and referenced in the academic literature, for example. Therefore, exposing information on discrepancies is an important outcome of building the OpenTrials database.
+
+#### How
+
+Use the [OpenTrials API](http://api.opentrials.net/v1/docs/) to find trials with detected discrepancies. Look for common types of discrepancies, and build basic visualisations based on type (e.g.: the publication date or participation count) or frequency (e.g. are discrepancies becoming more common or less common?).
+
+#### Skills
+
+- Intermediate JavaScript and HTML
+- Basic knowledge of visualisations
+
+### Visualise publication lag
+
+#### What
+
+It is imperative that results of clinical trials are published. This is how knowledge is shared: about the effectiveness, or not, of a particular intervention for a particular condition, for example. Unfortunately, many trials have their results published with a long delay, or, simply do not have their results published at all.
+
+#### How
+
+Use the [OpenTrials API](http://api.opentrials.net/v1/docs/) to find trials with a large lag in publication of results, or, trials that have finished and have not yet published results. Build basic visualisations that show lag over time, or lag by condition, by treatment, by trial sponsor, by primary investigator and so on.
```

Comment:
We don't have the date a trial's results were published in our DB, so this isn't possible yet.